sources/packages/web/src/services/http/ApplicationExceptionApi.ts

```diff
@@ -46,7 +46,7 @@ export class ApplicationExceptionApi extends HttpBaseClient {
   /**
    * Gets all pending student application exceptions.
    * @param paginationOptions options to execute the pagination.
-   * @returns list of student application exceptions.
+   * @returns list of student application appeals.
```

Comment:
It says "appeals" on a `getPendingExceptions`.

Reply:
I will update, thanks.